dataactcore/scripts/pullFPDSData.py

```diff
@@ -1210,6 +1211,9 @@ def process_and_add(data, contract_type, sess, sub_tier_list, county_by_name, co
                                    county_by_code=county_by_code, state_code_list=state_code_list,
                                    country_list=country_list)
             tmp_obj['updated_at'] = now
+
+            tmp_obj['business_categories'] = get_business_categories(row=tmp_obj, data_type='fpds')
```

Comment:
There is no reason to put this in two places. There is a function called `calculate_remaining_fields` in this file for this exact purpose so all derivations are in the same place.